test/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ if(ENABLE_DATA_TOOLS)
 endif()
 
 if(ENABLE_SERVICES)
-  list(APPEND tests loki_service skadi_service thor_service)
```

Comment:
the thor service test got disabled when we switched to completely using protobuf between the service layers